.eleventy.js

```diff
@@ -34,25 +34,39 @@ module.exports = function (config) {
     config.addTransform("parse", parseTransform);
 
     // Passthrough copy
+    config.addPassthroughCopy({"src/admin/config.yml": "admin/config.yml"});
     config.addPassthroughCopy({"src/assets/icons": "/"});
     config.addPassthroughCopy({"src/assets/images": "assets/images"});
     config.addPassthroughCopy({"src/posts/images": "posts/images"});
 
+    config.addNunjucksFilter("getSiblingValue", function (value, inputKey, inputValue, siblingKey) {
+        let siblingValue = false;
+        value.forEach(item => {
+            if (item[inputKey] === inputValue) {
+                siblingValue = item[siblingKey];
+            };
+        });
+
+        return siblingValue;
+    });
+
     const now = new Date();
 
     // Custom collections
     const livePosts = post => post.date <= now && !post.data.draft;
-    config.addCollection("posts", collection => {
-        return [
-            ...collection.getFilteredByGlob("./src/posts/*.md").filter(livePosts)
-        ];
-    });
-
-    // The following collection is used to create a collection of posts for the RSS feed.
-    config.addCollection("postFeed", collection => {
-        return [...collection.getFilteredByGlob("./src/posts/*.md").filter(livePosts)]
-            .reverse()
-            .slice(0, site.maxPostsInFeed);
+    siteConfig.languages.forEach(lang => {
+        config.addCollection(`posts_${lang.code}`, collection => {
+            return [
+                ...collection.getFilteredByGlob(`./src/collections/posts/${lang.code}/*.md`).filter(livePosts)
```

Comment:
Any particular reason to use the array + spread syntax here rather than returning the original array value?

Reply:
It was in the original code that I picked up from our GSOC contributor and I never thought to change it. And you're right, it's not necessary. I thought at first that `getFilteredByGlob` might be returning a Set rather than an array but I checked and that's not the case.